packages/react-admin/src/reducer/admin/auth.js

```diff
@@ -0,0 +1,67 @@
+import {
+    USER_CHECK_SUCCESS,
+    USER_LOGIN_SUCCESS,
+} from '../../actions/authActions';
+
+const initialState = {
+    user: null,
```

Comment:
I suggest we initialize it using localStorage, and use the saga to store the credentials there. What do you think ?

Reply:
We did not enforce how to store the authentication related until now. I'm still not sure we should.